src/bitmessageqt/__init__.py

```diff
@@ -3656,33 +3706,29 @@ def getAccountTextedit(self, account):
     def getCurrentSearchLine(self, currentIndex=None, retObj=False):
         if currentIndex is None:
             currentIndex = self.ui.tabWidget.currentIndex()
-        messagelistList = [
+        messagelistList = (
             self.ui.inboxSearchLineEdit,
             False,
             self.ui.inboxSearchLineEditSubscriptions,
             self.ui.inboxSearchLineEditChans,
-        ]
+        )
         if currentIndex >= 0 and currentIndex < len(messagelistList):
             if retObj:
                 return messagelistList[currentIndex]
             else:
-                return messagelistList[currentIndex].text().toUtf8().data()
-        else:
-            return None
+                return messagelistList[currentIndex].text().toUtf8()
```

Comment:
Have you tested this?

Reply:
oops! https://github.com/g1itch/PyBitmessage/commit/1eb3eb0ed5ab4f867abbad813fd9d6d70720d35a